Examples/DataRepresentation/Image/ImageAdaptor2.cxx

```diff
@@ -145,9 +146,7 @@ int main( int argc, char *argv[] )
 
   // Software Guide : BeginCodeSnippet
   using OutputImageType = itk::Image< unsigned char, Dimension >;
-  using RescalerType = itk::RescaleIntensityImageFilter<
-                                            ImageAdaptorType,
-                                            OutputImageType >;
+  using RescalerType = itk::RescaleIntensityImageFilter< ImageAdaptorType, OutputImageType >;
```

Comment:
Some of these might be manual edits to keep the lines from going off the page in LaTeX. Maybe we can add additional `.clang-format` file in the examples directory to treat `// Software Guide : BeginCodeSnippet` and `// Software Guide : EndCodeSnippet` as special format on/off "macros".

Reply:
This is an interesting idea.  Can we make the fixing of the software guide a separate issue to be followed up on after the initial commit?

Reply:
Created issue https://github.com/InsightSoftwareConsortium/ITK/issues/1195